modules/oadp-operator-issues.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-operator-issues_{context}"]
+= OADP Operator issues
+
+The {oadp-first} Operator might encounter issues caused by problems it is not able to resolve.
+
+[id="oadp-operator-fails-silently_{context}"]
+== OADP Operator fails silently
+
+The S3 buckets of an OADP Operator might be empty, but when you run the command `oc get po -n <OADP_Operator_namesapce>`, you see that the Operator has a status of `Running`.  In such a case, the Operator is said to have _failed silently_ because it incorrectly reports that it is running.
```

Comment:
Is there a typo in this command? (OADP_Operator_namespace)

Reply:
Not necessarily see https://github.com/openshift/origin/issues/5325

Reply:
I'm sorry, I don't see anything in the PR you linked above that related to my question, but if you're telling me that "namespace" should be spelled "namesapce" in the command, then I can merge it, I guess. Please confirm.

Reply:
![Screenshot 2023-11-21 at 1 56 18 PM](https://github.com/openshift/openshift-docs/assets/96192119/ef33cb02-5b0e-4bc1-ae17-886ac06c8447)

?

Reply:
@kcarmichael08 - apologies, i am an idiot. i thought you were looking at the `oc get po` command... which i double checked earlier.
Yes namespace.... double apologies for missing it

Reply:
No, it's ok, I was just confused and thought I was missing something! It's been a long (short) week! :)

Reply:
@kcarmichael08 Good catch. I'm the sleepyhead. Fixed now.